classes/PodsAdmin.php

```diff
@@ -2090,7 +2090,7 @@ public function admin_components () {
             if ( !empty( $component_data[ 'category' ] ) ) {
                 $category_url = pods_query_arg( array( 'view' => sanitize_title( $component_data[ 'category' ] ), 'pg' => '', 'page' => $_GET[ 'page' ] ) );
 
-                $component_data[ 'category' ] = '<a href="' . esc_url( $category_url ) . '">' . $component_data[ 'category' ] . '</a>';
+                $component_data[ 'category' ] = '<a href="' . esc_url( $category_url ) . '">' . __( $component_data[ 'category' ], 'pods' ) . '</a>';
```

Comment:
@pavelevap I've just reverted it in https://github.com/pods-framework/pods/pull/4001/commits/d688bf065f802ae4aba8a2f866af1f775768768b.